apps/events/models.py

```diff
@@ -11,9 +11,9 @@ def event_thumbmail_path(instance, filename):
 
 class Event(models.Model):
     title = models.CharField(max_length=100)
-    description = models.TextField(max_length=2500)
     schedule = models.DateTimeField()
     venue = models.CharField(max_length=50)
+    description = models.TextField(max_length=2500)
```

Comment:
Can you please remove this file from the PR? Just change back the position of description field.